version/version.go

```diff
@@ -58,7 +58,7 @@ func Get() Info {
 		GoVersion:       runtime.Version(),
 		Compiler:        runtime.Compiler,
 		Platform:        fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
-		AzureSdkVersion: fmt.Sprintf("v%s", azuresdkversion.Number),
```

Comment:
This was just a bug I noticed--`azuresdkversion.Number` is already "v55.2.0", so no there's no need to prepend a "v."